### PR TITLE
DOCS-5657 Handling Pop-Ups in Browser Tests Guide Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates the output for `prompt` and `confirm` modals and replaces the screenshot of the Advanced Options menu.

### Motivation

DOCS-5657

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/popup.md [hunk 1]
```diff
@@ -69,3 +70,4 @@ Alternatively, use one of these methods to ensure your pop-up is closed and your
 [2]: /synthetics/browser_tests/#test-configuration
 [3]: /synthetics/browser_tests/advanced_options/#optional-step
 [4]: /synthetics/browser_tests/actions#test-your-ui-with-custom-javascript
+[5]: /synthetics/browser_test
```
Comment (alai97) [inline content/en/synthetics/guide/popup.md:73]:
```suggestion
[5]: /synthetics/browser_tests
```

File: content/en/synthetics/guide/popup.md [hunk 2]
```diff
@@ -20,25 +20,26 @@ This guide describes how to manage pop-ups such as modals or application windows
 Synthetic browser tests automatically handle [JavaScript modals][1]:
 
  - `alert` modals are closed. 
- - `prompt` and `confirm` modals are answered `Lorem Ipsum`.
+ - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked.
```
Comment (alai97) [inline content/en/synthetics/guide/popup.md:23]:
```suggestion
 - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked. `prompt` modals offer a text input field that is filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
```

File: content/en/synthetics/guide/popup.md [hunk 3]
```diff
@@ -20,25 +20,26 @@ This guide describes how to manage pop-ups such as modals or application windows
 Synthetic browser tests automatically handle [JavaScript modals][1]:
 
  - `alert` modals are closed.
```
Comment (alai97) [inline content/en/synthetics/guide/popup.md:22]:
```suggestion
 - `alert` modals are immediately dismissed. 
```

File: content/en/synthetics/guide/popup.md [hunk 4]
```diff
@@ -20,25 +20,26 @@ This guide describes how to manage pop-ups such as modals or application windows
 Synthetic browser tests automatically handle [JavaScript modals][1]:
 
  - `alert` modals are closed. 
- - `prompt` and `confirm` modals are answered `Lorem Ipsum`.
+ - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked. `prompt` modals offer a text input field that is filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
+ - `confirm` modals are answered with `true` for OK and `false` if the Cancel button or `Esc` is clicked.
```
Comment (alai97) [inline content/en/synthetics/guide/popup.md:24]:
```suggestion
 - `confirm` modals that ask for confirmation are answered with `true` for OK and `false` if the Cancel button or `Esc` is clicked.
```

File: content/en/synthetics/guide/popup.md [hunk 5]
```diff
@@ -20,25 +20,26 @@ This guide describes how to manage pop-ups such as modals or application windows
 Synthetic browser tests automatically handle [JavaScript modals][1]:
 
  - `alert` modals are closed. 
- - `prompt` and `confirm` modals are answered `Lorem Ipsum`.
+ - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked. `prompt` modals offer a text input field that is filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
```
Comment (alai97) [inline content/en/synthetics/guide/popup.md:23]:
```suggestion
 - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked. `prompt` modals that offer a text input field are filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
```

File: content/en/synthetics/guide/popup.md [hunk 6]
```diff
@@ -2,43 +2,44 @@
...
 
 ### JavaScript
 
 Synthetic browser tests automatically handle [JavaScript modals][1]:
 
- - `alert` modals are closed. 
- - `prompt` and `confirm` modals are answered `Lorem Ipsum`.
+ - `alert` modals are immediately dismissed for OK. 
+ - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked. `prompt` modals that offer a text input field are filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
+ - `confirm` modals that ask for confirmation are answered with `true` for OK and `false` if the Cancel button or `Esc` is clicked.
```
Comment (HugoPct) [inline content/en/synthetics/guide/popup.md:24]:
```suggestion
 - `confirm` modals that ask for confirmation are accepted.
```

File: content/en/synthetics/guide/popup.md [hunk 7]
```diff
@@ -2,43 +2,44 @@
...
 ## Modals
 
 ### JavaScript
 
 Synthetic browser tests automatically handle [JavaScript modals][1]:
 
- - `alert` modals are closed. 
- - `prompt` and `confirm` modals are answered `Lorem Ipsum`.
+ - `alert` modals are immediately dismissed for OK. 
+ - `prompt` modals are answered with the text returned or `null` if the Cancel button or `Esc` is clicked. `prompt` modals that offer a text input field are filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
```
Comment (HugoPct) [inline content/en/synthetics/guide/popup.md:23]:
```suggestion
 - `prompt` modals are filled with `Lorem Ipsum` for tests on Google Chrome or Microsoft Edge.
```

**Review by HugoPct — CHANGES_REQUESTED** [review]
changed the 3 items following etienne's suggestion

File: content/en/synthetics/guide/popup.md [hunk 8]
```diff
@@ -2,43 +2,44 @@
...
+    tag: 'Blog'
+    text: 'User experience monitoring with Datadog Browser Tests'
+  - link: 'synthetics/browser_tests'
+    tag: 'Documentation'
+    text: 'Learn about Synthetic browser tests'
 ---
 ## Overview
 
-This guide describes how to manage pop-ups such as modals or application windows in a Synthetics browser test.
+This guide describes how to manage pop-ups such as modals or application windows in a Synthetics [browser test][5].
```
Comment (rtrieu) [inline content/en/synthetics/guide/popup.md:14]:
```suggestion
This guide describes how to manage pop-ups such as modals or application windows in a Synthetic [browser test][5].
```

**Review by rtrieu — APPROVED** [review]
Minor suggestion but otherwise looks good!

**Review by HugoPct — APPROVED** [review]

**Pull request merged**